.travis.yml

```diff
@@ -0,0 +1,18 @@
+sudo: required
+dist: trusty
+language: node_js
+cache: yarn
+node_js:
+- '8'
+- '10'
+before_install:
+# Use newer yarn than that pre-installed in the Travis image.
+- curl -sSfL https://yarnpkg.com/install.sh | bash
+- export PATH="$HOME/.yarn/bin:$PATH"
+install:
+# Remove --ignore-engines when this fixed:
+# https://github.com/anodynos/upath/issues/14
+- yarn install --frozen-lockfile --ignore-engines
```

Comment:
We can probably remove `--ignore-engines` now. The issue seems to have been resolved in https://github.com/anodynos/upath/pull/15.